gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -115,7 +116,9 @@
    * authority corresponding to that root path in the process.
    */
   public static final HadoopConfigurationProperty<Boolean> GCE_BUCKET_DELETE_ENABLE =
-      new HadoopConfigurationProperty<>("fs.gs.bucket.delete.enable", false);
+      new HadoopConfigurationProperty<>(
```

Comment:
Do we really wanna do this? This looks to me less obvious by using a default. I feel we better unify the way to define these default values, not mixed.